docs/OnTrack/Deployment/Google Cloud/Overview.md

```diff
@@ -0,0 +1,152 @@
+deployment.
+
+### Deploying OnTrack
+
+From here, we pulled down the doubtfire-deploy repository, generated a new certificate and private
+key for the host, and ran docker compose to deploy the containers for the OnTrack deployment.
+
+![gcp-instance-docker](assets/gcp-instance-docker.png "Instance running docker containers")
+
+And we can browse to OnTrack over the internet using our public IP address via HTTPS (port 443).
+
+![ontrack-login](assets/ontrack-login.png "OnTrack Login")
+
+## Next stages
+
+Since we have a functional and publicly accessible instance of Doubtfire running in GCP, the next
+stages would be to focus on the Deployment team project objectives, such as -
+
+- Create a CI/CD pipeline that automates the building, deployment, and validation of the Thoth Tech
+  OnTrack deployment onto GCP.
+- LDAP authentication for OnTrack.
+- Email notifications configured with an SMTP server.
+- Review security posture and instance sizing of the Thoth Tech OnTrack deployment in GCP.
+
+Here's a high-level diagram of using CI/CD pipeline to automate the deployment of OnTrack onto GCP -
+
+![Google Cloud - CI/CD](assets/GoogleCloud_CICD.jpg "Google Cloud - CICD")
+
+## Links
+
+- [GitHub repo for Thoth-Tech/Doubtfire-deploy-GCP](https://github.com/thoth-tech/doubtfire-deploy-GCP)
+- [GitHub repo for Thoth-Tech/Doubtfire-deploy/DEPLOYING.md](https://github.com/thoth-tech/doubtfire-deploy/blob/main/DEPLOYING.md)
+- [Trello board for Deployment - Google Cloud epic](https://trello.com/b/dI1yx9A1/deployment)
+- [Miro for Google Cloud - Doubtfire](https://miro.com/app/board/uXjVO0h8ZSE=/?share_link_id=62396987373)
+- [Miro for Google Cloud - CI/CD](https://miro.com/app/board/uXjVO64xoQw=/?share_link_id=57734801709)
+- [Pull Request - OnTrack deployment on GCP](https://github.com/thoth-tech/doubtfire-deploy-GCP/pull/5)
+
+## Assets
+
+- [Google Cloud - Doubtfire](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/GoogleCloud_Doubtfire.jpg)
+- [Google Cloud - CI/CD](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/GoogleCloud_CICD.jpg)
+- [doubtfire-localhost-compose](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/doubtfire-localhost-compose.png)
+- [doubtfire-localhost-docker](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/doubtfire-localhost-docker.png)
+- [doubtfire-localhost](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/doubtfire-localhost.png)
+- [gcp-instance-docker](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/gcp-instance-docker.png "Instance running docker containers")
+- [ontrack-login](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/ontrack-login.png "OnTrack Login")
```

Comment:
**suggestion:** These links are already placed throughout the document. I wonder if we can just drop this section as it creates unneeded maintainance costs. WDYT?

Reply:
Agreed. Sorted now.